### DISCUSSION TITLE
Flyway sqlMigrationSuffixes

### DISCUSSION THREAD

**Discussion by schltgn** [quarkusio/quarkus#43357] [Community]
Hi,
I am currently migrating an application to Quarkus in which we use Flyway. In the configuration we have used the parameter `sqlMigrationSuffixes` ([Documentation](https://documentation.red-gate.com/fd/sql-migration-suffixes-224919750.html)). However, I don't see any way to set this parameter in the Quarkus-Flyway integration.
Or am I missing a possibility to set this parameter somehow?

**Comment by mschorsch** [comment]
@schltgn I think this is not supported at the moment. I would recommend to open a [feature request](https://github.com/quarkusio/quarkus/issues/new?assignees=&labels=kind%2Fenhancement&projects=&template=feature_request.yml) in the Quarkus repo.
